packages/functions/src/load-balancing/selectBestRTDBInstance.ts

```diff
@@ -93,15 +94,15 @@ export const selectBestRTDBInstance = functions.region(
 
   await runTransactionWithRetry(db, async transaction => {
     const gameDoc = await transaction.get(gameRef);
```

Comment:
Add bestRTDBInstanceName = oldBestRTDBInstanceName to ensure if transaction retries?